lib/screens/login/login_form.dart

```diff
@@ -14,6 +14,7 @@ class LoginForm extends StatefulWidget {
 
 class _LoginFormState extends State<LoginForm> {
   final _formKey = GlobalKey<FormState>();
+  final _fieldSpacing = 20.0;
```

Comment:
@Thieurom By doing this way, we could not set with const of SizedBox. Could we bring it to the top of `login_form` and make it as const? 🙏

Reply:
Oh, Flutter/Dart seems to have some interesting conventions 😅, one of those is declaring global variables. I'll update accordingly 👍